src/redux/books/books.js

```diff
@@ -0,0 +1,32 @@
+const actions = {
+  ADD_BOOK: 'ADD_BOOK',
+  DELETE_BOOK: 'DELETE_BOOK',
```

Comment:
- [ ] Let's refactor these actions to follow [the ducks' pattern](https://medium.com/@matthew.holman/what-is-redux-ducks-46bcb1ad04b7).

  For example `ADD_BOOK: 'bookstore/books/ADD_BOOK'`